harness/src/test/java/org/jboss/pvt/harness/TestConfiguration.java

```diff
@@ -1,46 +1,93 @@
 package org.jboss.pvt.harness;
 
-import org.jboss.pvt.harness.configuration.DefaultConfiguration;
+import java.util.Properties;
+
+import org.jboss.pvt.harness.configuration.PVTConfiguration;
+import org.jboss.pvt.harness.exception.PVTSystemException;
+import org.jboss.pvt.harness.utils.HttpUtils;
+import org.jboss.pvt.harness.utils.ZipUtils;
 import org.junit.rules.TestRule;
 import org.junit.runner.Description;
 import org.junit.runners.model.Statement;
 
 import java.io.File;
+import java.io.FileReader;
 import java.util.Properties;
 
 /**
  * Created by rnc on 12/08/16.
  */
-public class TestConfiguration extends DefaultConfiguration implements TestRule
+public class TestConfiguration implements PVTConfiguration, TestRule
 {
```

Comment:
DefaultConfiguration call downloadZips in constructor, this causes exception in PVTConfiguration initializing.